templates/policy-controller.yaml

```diff
@@ -44,66 +54,110 @@ subjects:
   name: calico-kube-controllers
   namespace: kube-system
 ---
-# Include a clusterrole for the kube-controllers component,
-# and bind it to the calico-kube-controllers serviceaccount.
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: calico-kube-controllers
```

Comment:
Looks like there are two copies of the calico-kube-controllers Deployment in this template now. Can you remove one?

Reply:
Thanks for spotting that; completely missed it.